release-tools/cloudbuild.yaml

```diff
@@ -26,7 +26,7 @@ steps:
   # The image must contain bash and curl. Ideally it should also contain
   # the desired version of Go (currently defined in release-tools/prow.sh),
   # but that just speeds up the build and is not required.
-  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20240718-5ef92b5c36'
```

Comment:
could you fix the issue here: https://github.com/kubernetes-csi/csi-release-tools/blob/master/cloudbuild.yaml

Reply:
this repo just vendors that project